geopandas/plotting.py

```diff
@@ -53,58 +61,52 @@ def plot_polygon_collection(ax, geoms, colors_or_values, plot_values,
 
     geoms : a sequence of `N` Polygons and/or MultiPolygons (can be mixed)
 
-    colors_or_values : a sequence of `N` values or RGBA tuples
-        It should have 1:1 correspondence with the geometries (not their components).
+    values : a sequence of `N` values
+        Values will be mapped to colors using vmin/vmax/cmap. They should
+        have 1:1 correspondence with the geometries (not their components).
+        Otherwise follows `color` / `facecolor` kwargs.
+
+    edgecolor : single color or sequence of `N` colors
+        Color for the edge of the polygons
 
-    plot_values : bool
-        If True, `colors_or_values` is interpreted as a list of values, and will
-        be mapped to colors using vmin/vmax/cmap (which become required).
-        Otherwise `colors_or_values` is interpreted as a list of colors.
+    facecolor : single color or sequence of `N` colors
+        Color to fill the polygons. Cannot be used together with `values`.
+
+    color : single color or sequence of `N` colors
+        Sets both `edgecolor` and `facecolor`
+
+    **kwargs
+        Additional keyword arguments passed to the collection
 
     Returns
     -------
 
     collection : matplotlib.collections.Collection that was plotted
     """
-
     from descartes.patch import PolygonPatch
     from matplotlib.collections import PatchCollection
 
-    components, component_colors_or_values = _flatten_multi_geoms(
-        geoms, colors_or_values)
+    geoms, values = _flatten_multi_geoms(geoms, values)
```

Comment:
I think this is what you were referring to about the failing unit test that might become irrelevant if #318 decides to default plot uniform color? i.e. that we no longer flatten multi-geoms when passed a `list` of `color`?

I'll add my support for uniform color default.  To support the legacy rainbow default is why I had implemented this using `color_or_values`.